packages/modern-poalim-scraper/src/scrapers/hapoalim.ts

```diff
@@ -1,23 +1,27 @@
 import inquirer from 'inquirer';
 import type { Page } from 'puppeteer';
 import type { AccountDataSchema } from '../__generated__/accountDataSchema.js';
-import type { ForeignSwiftTransaction } from '../__generated__/foreignSwiftTransaction.js';
-import type { ForeignSwiftTransactions } from '../__generated__/foreignSwiftTransactions.js';
 import type { ForeignTransactionsBusinessSchema } from '../__generated__/foreignTransactionsBusinessSchema.js';
 import type { ForeignTransactionsPersonalSchema } from '../__generated__/foreignTransactionsPersonalSchema.js';
 import type { HapoalimDepositsSchema } from '../__generated__/hapoalimDepositsSchema.js';
 import type { HapoalimForeignDepositsSchema } from '../__generated__/hapoalimForeignDepositsSchema.js';
 import type { ILSCheckingTransactionsDataSchema } from '../__generated__/ILSCheckingTransactionsDataSchema.js';
 import accountDataSchemaFile from '../schemas/accountDataSchema.json' with { type: 'json' };
-import foreignSwiftTransactionSchema from '../schemas/foreignSwiftTransaction.json' with { type: 'json' };
-import foreignSwiftTransactionsSchema from '../schemas/foreignSwiftTransactions.json' with { type: 'json' };
 import foreignTransactionsBusinessSchema from '../schemas/foreignTransactionsBusinessSchema.json' with { type: 'json' };
 import foreignTransactionsPersonalSchema from '../schemas/foreignTransactionsPersonalSchema.json' with { type: 'json' };
 import depositsSchema from '../schemas/hapoalimDepositsSchema.json' with { type: 'json' };
 import hapoalimForeignDepositsSchema from '../schemas/hapoalimForeignDepositsSchema.json' with { type: 'json' };
 import ILSCheckingTransactionsDataSchemaFile from '../schemas/ILSCheckingTransactionsDataSchema.json' with { type: 'json' };
 import { fetchGetWithinPage, fetchPoalimXSRFWithinPage } from '../utils/fetch.js';
 import { validateSchema } from '../utils/validate-schema.js';
```

Comment:
so in the future, we would replace validate schema completely with parse from Zod?